File: CS/WindowsApplication204/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WindowsApplication204
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
        }

        public object DataSource
        {
            get { return dataLayoutControl1.DataSource; }
            set { dataLayoutControl1.DataSource = value; }
        }
    }
}

File: VB/WindowsApplication204/Form2.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace WindowsApplication204
	Partial Public Class Form2
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Public Property DataSource() As Object
			Get
				Return dataLayoutControl1.DataSource
			End Get
			Set(ByVal value As Object)
				dataLayoutControl1.DataSource = value
			End Set
		End Property
	End Class
End Namespace